checkstyle-tester/README.md

```diff
@@ -19,7 +19,7 @@ Depending on the type of external projects you wish to generate reports for, you
 
 **baseBranch** (b) - name of the base branch in local Checkstyle repository (optional, if asent, then the tool will use only patchBranch in case the tool mode is 'single', otherwise baseBrach will be set to 'master');
 
-**mode** (m) - the mode of the tool: 'single' or 'diff' (optional, default is 'diff'. Set this option to 'single' if your patch branch contains changes for any option that can't work on master/base branch. 
+**mode** (m) - the mode of the tool: 'single' or 'diff' (optional, default is 'diff'. Set this option to 'single' if your patch branch contains changes for any option that can't work on master/base branch.
```

Comment:
Make a separate PR for these fixes so we can merge right away.

Reply:
Done.